### DISCUSSION TITLE
DocTR for Indic Language

### DISCUSSION THREAD

**Discussion by PrabhatGyan** [mindee/doctr#1680] [Q&A]
Does docTR at present support Indic Languages? Especially Hindi.
If yes, can someone please share a sample code. 
Appreciate the help.

**Comment by felixdittrich92** [comment]
Hi @PrabhatGyan :wave:,

docTR doesn't provides support for indic vocabs actually.
But you could try to train your own model/s:

https://mindee.github.io/doctr/using_doctr/custom_models_training.html
